web/src/components/layout/header/Navbar.vue

```diff
@@ -1,39 +1,39 @@
 <template>
   <nav
-    class="flex bg-wp-primary-200 dark:bg-wp-primary-300 text-neutral-content p-4 border-b border-wp-background-100 font-bold text-wp-primary-text-100"
+    class="flex border-wp-background-100 bg-wp-primary-200 dark:bg-wp-primary-300 p-4 border-b font-bold text-neutral-content text-wp-primary-text-100"
   >
     <div class="flex items-center space-x-2">
       <router-link :to="{ name: 'home' }" class="flex flex-col -my-2 px-2">
         <WoodpeckerLogo class="w-8 h-8" />
         <span class="text-xs" :title="version?.current">{{ version?.currentShort }}</span>
       </router-link>
-      <router-link v-if="user" :to="{ name: 'repos' }" class="navbar-link navbar-clickable">
+      <router-link v-if="user" :to="{ name: 'repos' }" class="navbar-clickable navbar-link">
         <span class="flex md:hidden">{{ $t('repos') }}</span>
-        <span class="hidden md:flex">{{ $t('repositories') }}</span>
+        <span class="md:flex hidden">{{ $t('repositories') }}</span>
       </router-link>
-      <a href="https://woodpecker-ci.org/" target="_blank" class="navbar-link navbar-clickable hidden md:flex">{{
+      <a href="https://woodpecker-ci.org/" target="_blank" class="md:flex hidden navbar-clickable navbar-link">{{
         $t('docs')
       }}</a>
-      <a v-if="enableSwagger" :href="apiUrl" target="_blank" class="navbar-link navbar-clickable hidden md:flex">{{
+      <a v-if="enableSwagger" :href="apiUrl" target="_blank" class="md:flex hidden navbar-clickable navbar-link">{{
         $t('api')
       }}</a>
     </div>
-    <div class="flex ml-auto -m-1.5 items-center space-x-2">
+    <div class="flex items-center space-x-2 -m-1.5 ml-auto">
       <IconButton
         v-if="user?.admin"
-        class="navbar-icon relative"
+        class="relative navbar-icon"
         :title="$t('settings')"
         :to="{ name: 'admin-settings' }"
       >
         <Icon name="settings" />
         <div
           v-if="version?.needsUpdate"
-          class="absolute top-2 right-2 bg-int-wp-state-error-100 rounded-full w-3 h-3"
+          class="top-[0.15rem] right-[0.4rem] absolute bg-int-wp-state-error-100 rounded-full w-3 h-3"
```

Comment:
I know It's still a draft (let me know if you don't want to have feedback during in draft state) but I'm against such micro-changes. On the one side you would like to generalize things and in this place you switch to custom values to micro-optimize things.

Reply:
Not sure if I like the overall direction 🙈

Reply:
You said before you don't want half-baked solutions for UI, i.e. only changing the relative icon size 🙃️

Hence I am now trying to achieve matching overall picture that works OOB, including aligning icons in height that had a random height before (e.g. in the navbar, the admin settings icon is not aligned with the two ones to its right).

Remember that I also adjusted the icons in size again to match your opinion, not mine ;)

Please have some patience, I'll ping you again once I have something ready "to show".

Reply:
> You said before you don't want half-baked solutions for UI, i.e. only changing the relative icon size 🙃️

:D ok, point for you, however let's try to work together (I know I can also still improve on this point)

Regarding the Icon, I don't think we need to change it at all. It now looks pretty big. Why do you try to make the icon of the icon button the same size as the avatar? The entire button should match the size, but not the icon:

<img width="246" alt="grafik" src="https://github.com/user-attachments/assets/2563c834-ddbc-43d8-a70e-da1237b1e7cc" />

Reply:
Because there is no adjacent component next to it matching the size. And it that case it looks non-harmonic not having a match with the other two components right next to it. 

Yes, it matches with the line height on the other side. But that is "the other side". One option could also be to move it to the left then. 

I somewhat agree with the overall size though I almost got used to it already in the instance I am running this image on. If this would have been the standard until now, you likely wouldn't have noticed 🙃

Reply:
Ok, for me it's way too big, but I'm ok to keep it that way.

> One option could also be to move it to the left then.

What do you mean with this?

Reply:
Move the admin settings icon to the left of the navbar so it matches visually with the line height and the existing text.

Reply:
Ok, definitely don't want to move it to the left. If that's the only alternative, just keep the current approach.